tests/app_loading/tests.py

```diff
@@ -75,3 +78,23 @@ def test_get_models_only_returns_installed_models(self):
         self.assertNotIn(
             "NotInstalledModel",
             [m.__name__ for m in apps.get_models()])
+
+    def test_declare_model_outside_app(self):
```

Comment:
Sorry to be picky, could you rename the test function name to something a bit more verbose? a suggestion could be: "test_exception_raised_if_model_declared_outside_app"